public/components/marketing-testimonials/9.html

```diff
@@ -0,0 +1,56 @@
+<div class="my-0 mx-auto flex flex-wrap items-center justify-center">
+
+    <!-- Card 1 -->
+    <section class="w-80 rounded-sm shadow-lg bg-slate-900 px-10 py-8 text-white m-2 ">
+        <div class="flex flex-col items-center text-center">
```

Comment:
Don't need `flex flex-col items-center`.

Reply:
removing `flex flex-col items-center` will change the layout..any alternative to this? I guess `flex` is fine

Reply:
Nah, it would be fine as you are using `block` layout elements.